src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCodeEqualsParameter.java

```diff
@@ -7,7 +7,7 @@
 package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
 
 public class InputEqualsHashCodeEqualsParameter {
-    public static class TestClass1 { // ok
+    public static class TestClass1 {
```

Comment:
The files lgtm....

All your changes must strictly stick to one commit.
Squash and rebase to single commit when necessary.

It is important to understand what the Issue demands before you jump into it....
Please go through the Issue https://github.com/checkstyle/checkstyle/issues/13213 description before sending PR.
You must also remove the respective suppression from [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml)